docs/README.md

```diff
@@ -0,0 +1,38 @@
+
+# Documentation
+
+Welcome to the `kube-linter` documentation. Read on for more detailed information about using and configuring the tool.
+
+## Exporing the CLI
+
+You can run `kube-linter --help` to see a list of supported commands and flags. For each subcommand, you can
+run `kube-linter <subcommand> --help` to see detailed help text and flags for it.
+
+## Running the linter
+
+To lint directories or files, simply run `./kube-linter lint files_or_dirs ...`. If a directory is passed, all files
+with `.yaml` or `.yml` extensions are parsed, and Kubernetes objects are loaded from them. If a file is passed,
+it is parsed irrespective of extension.
+
+Users can pass a config file using the `--config` file to control which checks are executed, and to configure custom checks.
```

Comment:
flag again (is there some way to not duplicate the contents between this and the top-level readme)?

Reply:
Fixed. And I just removed the duplicate content from the top-level README in favour of just linking.